Bottom: 7fe0e6831d0ae8e498fd1e73742343a59e98d933
Top:    28106fb646910cb6b7b489a0ea9bec614c326d40
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-16 00:03:39 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
